### ISSUE TITLE
Dispersion model backward run example?

### ISSUE THREAD

**Issue by agdhruv** [rich-iannone/splitr#66]
Hi, thanks for this package!

I've been tinkering around with it for a few days but haven't been able to run a backward dispersion model. Could you please post a short example that works?

**Comment by AlFontal** [comment]
I am finding the same problem. I know this is over two years later, but were you able to work it out?

**Comment by agdhruv** [comment]
Nope, wasn't able to figure this out. I ended up using something different.

**Comment by AlFontal** [comment]
I guessed so. Backward dispersion is working in my fork now [AlFontal/splitr@main](https://github.com/AlFontal/splitr). But I can imagine you are not in need for this anymore after two years 😆
